index.js

```diff
@@ -1,11 +1,41 @@
-var oldHello = function(name) {
-  console.log('hello, ' + name + '!');
-};
+// var oldHello = function(name) {
+//   console.log('hello, ' + name + '!');
+// };
 
 
-const hello = (name) => {
-  console.log(`hello, ${name}!`);
-};
+// const hello = (name) => {
+//   console.log(`hello, ${name}!`);
+// };
 
-oldHello('world');
-hello('es6');
+// oldHello('world');
+// hello('es6');
+
+function printNames(){
+  var names = [ "Andy Mention",
+                "Emily Davis",
+                "Eric Fransen",
+                "Jessica Goulding",
+                "Jonmichael Chambers",
+                "Marc Garreau",
+                "Tan Doan",
+                "Alan Smith",
+                "Allison Larson",
+                "Andrew Watkins",
+                "Chad Brading",
+                "Corey Davis",
+                "Gustavo Villagrana",
+                "Hilary Denton",
+                "Horacio Chavez",
+                "Tim Proctor",
+                "Will Faurot" ]
+
+  var arrayLength = names.length;
+
+  for (var i = 0; i < arrayLength; i++) {
+    var currentName = names[i];
+    var div = document.createElement("div");
```

Comment:
Cool. So, you're using pure JS here, which is fine. You can also use jQuery if you want. It has a really nice `append` method for this sort of thing. http://api.jquery.com/append/